packages/next/pages/_document.tsx

```diff
@@ -761,6 +755,18 @@ export class NextScript extends Component<OriginProps> {
         {appScript}
         {staticMarkup ? null : this.getDynamicChunks()}
         {staticMarkup ? null : this.getScripts()}
+        {process.env.__NEXT_GRANULAR_CHUNKS ? (
```

Comment:
Is this the meat of the PR? Adding a separate async for this specific script to the end?

Also, we need to support the modern version of this.